src/monodroid/jni/dylib-mono.c

```diff
@@ -33,25 +33,18 @@ void monodroid_dylib_mono_free (struct DylibMono *mono_imports)
 	free (mono_imports);
 }
 
-int monodroid_dylib_mono_init (struct DylibMono *mono_imports, const char *libmono_path)
+int monodroid_dylib_mono_init (struct DylibMono *mono_imports, void *libmono_handle)
```

Comment:
Ops, this API change breaks Java.Interop here https://github.com/xamarin/java.interop/blob/master/src/java-interop/java-interop-gc-bridge-mono.c#L266

Reply:
I can update Java.Interop. Wonder whether we consider libmonodroid API as public, because it can break someone else too?

Reply:
When we migrate to C++ there will be more changes here. That code becomes a class and I added `extern "C"` versions of the functions - we can standarize/stabilize the API after the migration.